.circleci/config.yml

```diff
@@ -210,6 +210,18 @@ jobs:
           path: ~/project
       - store_test_results:
           path: ~/project
+  grace_daily:
+    machine: true
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run:
+          command: ./bin/linux/influxd --store=memory --log-level=debug
+          background: true
+      - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/grace/test-results/grace-results quay.io/influxdb/grace:daily
+      - store_artifacts:
+          path: ~/project
```

Comment:
Should we add this block:
```yaml
- store_test_results:
    path: ~/project
```
To match `grace_nightly`?

Reply:
Actually we should remove the `store_test_results:` from grace_nightly.The results are being uploaded by `store_artifact` step